docker/docker-compose.yaml

```diff
@@ -14,6 +14,8 @@ services:
       --keyper-set-manager-address ${KEYPER_SET_MANAGER_CONTRACT_ADDRESS}
       --rpc-url ${UPSTREAM_RPC}
       --delay-in-seconds ${DELAY_IN_SECONDS}
+    networks:
+      - app-network
```

Comment:
This isn't necessary. Compose creates a default network that all containers of a project are connected to by default.